credentials/src/main/kotlin/web5/sdk/credentials/PresentationExchange.kt

```diff
@@ -98,7 +101,7 @@ public object PresentationExchange {
     val vc = JWTParser.parse(vcJwt) as SignedJWT
 
     val vcPayloadJson = JsonPath.parse(vc.payload.toString())
-      ?: throw PresentationExchangeError("Failed to parse VC payload as JSON.")
+      ?: throw PresentationExchangeException("Failed to parse VC payload as JSON.")
```

Comment:
would love to know how to write a test that will generate a wonky vc payload that can't be parsed correctly and throws this new exception

Reply:
ah right. i just used the example jwt in jwt.io but got rid of the payload section and that seemed to do the trick (since this line is trying to parse the vc.payload)